README.md

```diff
@@ -7,6 +7,12 @@ Disassortative graphs refer to those with a low node homophily. In disassortativ
 
 **Please feel free to submit a pull request if you want to add good papers.**
 
+2022
+----
+* [AAAI 2022] **Graph Neural Networks with Heterophily** [[Paper]](https://arxiv.org/abs/2112.13562) 
```

Comment:
Sorry for the mistake here, the paper name should be Powerful Graph Convolutional Networks with Adaptive Propagation Mechanism for Homophily and Heterophily

Reply:
No Problem. I will check and fix it. Thank you for your PR.